lib/health.js

```diff
@@ -170,6 +170,53 @@ const cmdExec = (cmd, callback) => {
   });
 };
 
+const getSmartStatusOfDrive = (driveName, callback) => {
+  const cmd = `smartctl -H /dev/${driveName}`;
+  api.cp.exec(cmd, (error, stdout, stderr) => {
+    if (error) {
+      callback(error);
+      return;
+    }
+
+    if (stderr) {
+      callback(stderr);
```

Comment:
`stderr` is a string here, thus you should either log it here or create an error based on it, not pass it to the callback